src/main/java/millom/sandbox/pojos/Weather.java

```diff
@@ -0,0 +1,41 @@
+package millom.sandbox.pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Weather {
+  @JsonProperty("descriptions")
+  private Descriptions descriptions;
+  @JsonProperty("soles")
+  private List<Sol> soles;
+  private Weather(){
+  }
+
+  private Weather(WeatherBuilder weatherBuilder){
+    this.descriptions = weatherBuilder.descriptions;
+    this.soles = weatherBuilder.soles;
+  }
+
+  public Descriptions getDescriptions() {
+    return descriptions;
+  }
+
+  public List<Sol> getSoles() {
+    return soles;
+  }
+
+  public static class WeatherBuilder {
+    // builder code
```

Comment:
I believe this comment states the obvious. Unnecessary comments make the code longer and more difficult to read. I'd suggest to avoid that.

Reply:
Sure. Removed comment.